src/styles/components/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  font-family: ' Roboto ', sans-serif;
```

Comment:
Font family names containing whitespace should be quoted. If a font family name is a keyword of CSS like 'inherit', 'serif', 'sans-serif', 'monospace', 'fantasy', or 'cursive', it must be quoted to prevent confusion with the keywords with the same names. The font family 'Roboto' should be quoted.